modules/demo_account/services.tf

```
@@ -9,7 +9,8 @@ locals {
 }
 
 module "services_from_csv" {
-  for_each = { for svc in local.csv_services : svc.service_name => svc }
+  depends_on = [module.teams_from_csv]
+  for_each   = { for svc in local.csv_services : svc.service_name => svc }
```

```suggestion
  for_each   = { for svc in local.csv_services : svc.service_name => svc }
```

I'm pretty sure this isn't needed because its used in `owner` field so terraform can figure it out